test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -980,40 +967,17 @@ UCS_TEST_P(test_ucp_mmap_atomic, reuse_buffer)
 
 UCP_INSTANTIATE_TEST_CASE_GPU_AWARE(test_ucp_mmap_atomic)
 
-class test_ucp_rkey_compare : public test_ucp_mmap {
-public:
-    void init() override
-    {
-        const size_t count = 5;
-        test_ucp_mmap::init();
-
-        for (int i = 0; i < count; i++) {
-            m_chunks.emplace_back(new mem_chunk(sender().ucph()));
-        }
-    }
-
-    void cleanup() override
-    {
-        m_chunks.clear();
-
-        test_ucp_mmap::cleanup();
-    }
-
-protected:
-    struct mem_chunk {
-        ucp_context_h           context;
-        ucp_mem_h               memh;
-        std::vector<ucp_rkey_h> rkeys;
-
-        mem_chunk(ucp_context_h);
-        ~mem_chunk();
-        ucp_rkey_h              unpack(ucp_ep_h, ucp_md_map_t md_map = 0);
-    };
+struct mem_chunk {
```

Comment:
maybe move mem_chunk to test_ucp_mmap, so it will not be in top-level namespace?